### DISCUSSION TITLE
Can not insert assets on pages

### DISCUSSION THREAD

**Discussion by VN1977** [requarks/wiki#4892] [Error / Bug Report]
Hi,
We use the last version 2.5.268 and the problem is that assets can upload only admin. 
I added permission and page rules for users group but users can only create a new folder but don't see it.
I even added page rule for /root folder but nothing helps.
![изображение](https://user-images.githubusercontent.com/25663601/149514576-50602069-c3a6-4bea-9645-25f117e9d948.png)

**Comment by KeremKoksal** [comment]
We have the same issue. 

We have created a groups for every department. If we create the folders via default wiki administrator, there are no problems but if the department user creates the folder, department user can not see this folder. 

![](https://user-images.githubusercontent.com/25663601/149514576-50602069-c3a6-4bea-9645-25f117e9d948.png)
i.e: if we create the folder named 'department-it' via Administrator (default wiki admin) there are no problems. Users that are in the 'department-it' group can see and upload to this folder.

But, if we didn't create the folder via administrator and a user from the 'department-it' group creates it, user can't see the folder on their upload page.
